### ISSUE TITLE
Templates not re-rendering in 2.3.6

### ISSUE THREAD

**Issue by edcoffin** [playframework/playframework#3608]
Greetings,

I'm using Play 2.3.6 on the Vanilla Activator Scala/Play project. Modifying a template file while the server is running does not update the browser after a refresh.

While the templates do compile [visible on the CL when using ~run from SBT], there are no effects visible in the browser. I can even load the URL in a separate browser and the page still loads in it's old state. Not until I restart the Play server (ctrl+d) or stop and start sbt again does the page show the correct template contents.

I've been using Play for 11 months and have never noticed anything like this.

Any thoughts?

**Comment by pjean** [comment]
Hi,

I'm using Play2.4 (sbt-plugin: 2.4.1, sbt: 0.13.8, java: 1.8.0-b132, os: Mac OS X 10.10.3) and I have the same problem. It seems that playWatchService has been replaced by fileWatchService and thus I tried to add the following to my build but without success.
```
PlayKeys.fileWatchService := play.runsupport.FileWatchService.sbt(pollInterval.value)
```
Any thoughts?

**Comment by jasongonzales23** [comment]
+1

I am on Play 2.4.2 and at first changes to my templates were being refreshed, but now I cannot them to refresh even if I completely restart with `activator run`.

At this point, I'd even take a workaround.

**Comment by nafg** [comment]
Can you create a minimal project that reproduces the issue?

**Comment by edcoffin** [comment]
The fix for me was to move all my projects to a new directory (I'm using OS X).

At one time I had a capital letter in source file path (e.g. `/Users/eric/Code/company/project1/.git`). I migrated this directory to a system (with a newer version of OS X) and replaced `Code` with `code`. It was around this time that I had problems with file watch.

The fix was to move my company project directory out of the `(c|C)ode` directory entirely ~> `/Users/eric/company/project1/.git` 

I haven't run into the problem since.

**Comment by toschill** [comment]
I have had the same issue. No auto-reloading. Edcoffins hint did the trick for me. I moved the whole project from my dev-folder `/Users/toschill/Documents/workspace/Studium/TeamprojektMSI/project1` to my user-folder `/Users/toschill/`. In the user-folder the auto-reloading works fine. Strangely I have recognized that when moving the folder back to the dev-folder it is still working. In my opinion the solution is not related to the capital letters but to the move-operation itself.

**Comment by piercelamb** [comment]
I'm having the same issue. I "forked" a play project that was working fine and had a folder structure like this:

    /My_Projects/landingpage/

to

    /My_Projects/experiments/intermediatepage

After reading the comments here, I moved intermediate page back to /My_Projects/, but auto-reloading still isn't working

**Comment by piercelamb** [comment]
Oddly enough, deleting that existing project, cloning the original from github into a new dir and letting intellij go through the whole setup process got auto-reload to work again.

**Comment by slohr** [comment]
activator 1.3.6
play 2.4.3
OSX 10.10.5
ITerm 1.0.0

Alas, none of the other fixes worked for me.

However, setting this option in my build.sbt file magically did:
fork in run := false

**Comment by piercelamb** [comment]
If you're using play 2.6.6 and sbt 1+ and encountering this issue, rollback your sbt version to 0.13.16 and it fixes the issue. (do this in project/build.properties)

**Comment by marcospereira** [comment]
@piercelamb this is probably #7898. See comments there about how to use a snapshot version to verify this was fixed for you.

**Comment by mnmami** [comment]
How odd?! in case someone slid into this situation as I did. I imported the project using Intellij. What happened next is that the views render (after I refresh the page) only upon the first change, any subsequent changes are not captured, until I interrupt the application (Ctrl-c) and 'sbt run' it again (same with ~run). I can see the changes saved to file when I open it using another text editor. 

However, use another text editor, like Gedit or Atom, work just fine.
